src/services/point.service.ts

```diff
@@ -15,8 +16,39 @@ export class PointService {
       return;
     }
 
+    // cache the #1 point holder before points are awarded
+    const [kingBeforeAward] = await this.getTopPoints(1);
+
     const userDoc = await this.getUserPointDoc(id);
     await PointsModel.updateOne(userDoc, { $inc: { numPoints: amount } });
+
+    const [kingAfterAward] = await this.getTopPoints(1);
+
+    // the king has changed!
+    if (kingBeforeAward.userID !== kingAfterAward.userID) {
+      // remove the role from the previous king
+      await this._guild.members.cache
+        .get(kingBeforeAward.userID)
+        ?.roles.remove(Constants.Roles.TacoKing);
+
+      // give the role to the new king
+      await this._guild.members.cache
+        .get(kingAfterAward.userID)
+        ?.roles.add(Constants.Roles.TacoKing);
+
+      // get the games channel
+      const gamesChan = this._guild.channels.cache.find(
+        (c) => c.name === Constants.Channels.Public.Games
+      );
+
+      if (!gamesChan) {
+        return;
+      }
```

Comment:
```suggestion
const gameChan = this._guildService.getChannel(Constants.Channels.Public.Games);
````

This method also asserts that the channel exists so no need for the null check either